backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -469,8 +491,15 @@ public FacilityStats getFacilityStats(@Argument UUID facilityId) {
         this.getFacilityById(facilityId)
             .orElseThrow(() -> new IllegalGraphqlArgumentException("Facility not found."));
 
+    Integer userWithSingleFacilityAccess;
```

Comment:
nit: should this be users plural? Same with the method `dbAuthorizationService.getUserWithSingleFacilityAccessCount`?

Reply:
OO nice catch! I'll address that!

Reply:
Done!